app/schemas/pagination_schema.js

```diff
@@ -0,0 +1,20 @@
+exports.paginationSchema = {
+  page: {
+    in: ['query'],
+    customSanitizer: {
+      options: value => (value < 1 || !value ? 1 : value)
+    },
+    isInt: {
+      errorMessage: 'must_be_int'
+    }
+  },
+  per_page: {
+    in: ['query'],
+    customSanitizer: {
+      options: value => (value < 1 || !value ? 10 : value)
+    },
+    isInt: {
+      errorMessage: 'must_be_int'
+    }
+  }
+};
```

Comment:
If I get it right, you are setting the defaults for pagination here. It would be better if those values are configurable, using the config file.